mobile/native/src/trade/order/mod.rs

```diff
@@ -22,14 +22,20 @@ pub enum OrderType {
 /// Internal type so we still have Copy on order
 #[derive(Debug, Clone, Copy)]
 pub enum FailureReason {
+    /// An error occurred when setting the Order to filling in our DB
     FailedToSetToFilling,
+    /// The order failed because we failed sending the trade request
     TradeRequest,
+    /// A failure happened during the initial phase of the protocol. I.e. after sending the trade
+    /// request
     TradeResponse,
     NodeAccess,
     NoUsableChannel,
-    ProposeDlcChannel,
+    /// The order failed due to collaboratively reverting the position
+    CollabRevert,
```

Comment:
🤔 That's probably not the reason why the order failed though. We can say that the _channel_ was collaboratively reverted before the order was `Filled`, but it's very likely that the order didn't get filled for a different reason.

I would adapt the docs, the name is okay though.

Reply:
This was my understanding though, i.e. we fail the order because we collaboratively reverted before it was filled

Reply:
Fair enough, I think you are right.